charts/sentry/templates/sentry/worker/deployment-sentry-worker-events.yaml

```diff
@@ -76,10 +76,18 @@ spec:
       - name: {{ .Chart.Name }}-worker
         image: "{{ template "sentry.image" . }}"
         imagePullPolicy: {{ default "IfNotPresent" .Values.images.sentry.pullPolicy }}
-        command: ["sentry"]
+        command: ["/bin/bash", "-c"]
         args:
-          - "run"
-          - "worker"
+          - |
+            {{- if .Values.sentry.worker.installAdditionalPackages }}
```

Comment:
Installing packages at runtime is not the best practice and will take much longer at startup compared to a custom image. Is this really necessary?

Reply:
Yes, I don't see any other options

Reply:
It seems to me that this can be solved in a more universal way: add the ability to set a custom command to launch the container. Then you can add what you need in your case

Reply:
Can you create new MR?

Reply:
@adonskoy Can you create new MR with you idea?